recipes/psychopy/meta.yaml

```diff
@@ -0,0 +1,98 @@
+{% set name = "PsychoPy" %}
+{% set version = "3.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: 5df7c0ff494368fcc4ade820d5b83135f82a44740868ed199d1023e181c30f95
+
+build:
+  number: 0
+  skip: true  # [py<36]
+  script: "{{ PYTHON }} -m pip install --no-deps . -vv"
+  entry_points:
+    - psychopy = psychopy.app.psychopyApp:main
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - arabic_reshaper
+    - astunparse
+    - configobj
+    - esprima-python
+    - freetype-py
+    - future
+    - gevent
+    - gitpython
+    - greenlet
+    - hdf5
+    - imageio >=2.5
+    - imageio-ffmpeg
+    - json_tricks
+    - matplotlib
+    - moviepy
+    - msgpack-numpy
+    - msgpack-python
+    - numpy
+    - opencv
+    - openpyxl
+    - pandas
+    - pillow
+    - psutil
+    - pyglet >=1.3
+    - pyobjc-framework-quartz  # [osx]
+    - pyopengl
+    - pyosf
+    - pyqmix >=2018.12.13  # [not linux]
+    - pyqt
+    - pyserial
+    - pysoundfile  # [not win]
+    - pytables
+    - python-bidi
+    - python-gitlab
+    - python-sounddevice
+    - pywin32  # [win]
+    - pyWinhook  # [win]
+    - pyyaml
+    - pyzmq
+    - requests
+    - scipy
+    - wxpython !=4.0.2, !=4.0.3
+    - xlrd
+
+test:
+  files:
+    - run_tests.py
+  requires:
+    - libglu  # [linux]
+  commands:
+    - psychopy --version
+    - python run_tests.py  # [not linux]
```

Comment:
Can you add a comment here about they the tests are being skipped? Might be worth including a link to this PR.

Reply:
There's already a comment in `run_tests.py` with a reference to this PR. I've now pushed a new. commit that adds more information on why we chose this particular test setup (and why we're skipping tests on Linux) directly to `meta.yaml`.